galsim/vonkarman.py

```diff
@@ -0,0 +1,183 @@
+# Copyright (c) 2012-2017 by the GalSim developers team on GitHub
```

Comment:
Are we going to do a 2017->2018 switch at some point?  (Preferably before 2019, but doesn't have to be in this PR.  Open new issue reminding Mike to use his magic script?)

Reply:
#944